src/renderer.h

```diff
@@ -255,6 +258,10 @@ class Renderer {
     bgfx::TextureHandle shadowMapTexture = BGFX_INVALID_HANDLE;
     bgfx::FrameBufferHandle shadowMapFrameBuffer = BGFX_INVALID_HANDLE;
 
+    bgfx::TextureHandle specularIblTexture = BGFX_INVALID_HANDLE;
+    bgfx::TextureHandle diffuseIblTexture = BGFX_INVALID_HANDLE;
+    bgfx::TextureHandle envBrdfTexture = BGFX_INVALID_HANDLE;

```

Comment:
Are those textures part of a single "draw" call logic on the GPU? Can we put them for eg. inside of `internalState.texHandlers` ?

Reply:
No, these textures are not part of a single draw call, they are global (common to all draw calls).